code/extensions/FluentExtension.php

```diff
@@ -493,21 +505,20 @@ public static function set_enable_write_augmentation($enabled)
      *
      * @param string $condition Condition SQL string
      * @param array $includedTables
-     * @param string $locale Locale to localise to
-     * @return string Column identifier in "table"."column" format if it exists in this condition
+     * @return array Array with items [$table, $column], or null if not found
```

Comment:
So `@return array|null` then? Or return an empty array for consistency? Bloody nuills...